src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costOfDay = 40;
+  const off7 = 50;
+  const off3 = 20;
```

Comment:
```suggestion
  const off7 = 50;
  const off3 = 20;
```
What does it mean? Let's use full, descriptive names